services/workflows-service/src/prisma/prisma.read-replica.service.ts

```diff
@@ -0,0 +1,39 @@
+import { readReplicas } from '@prisma/extension-read-replicas';
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from './prisma.service';
+import { AppLoggerService } from '@/common/app-logger/app-logger.service';
+import { PrismaClient } from '@prisma/client';
+import { env } from '@/env';
+
+@Injectable()
+export class ReadPrismaService extends PrismaService {
```

Comment:
Can you give an example of how will you use it? Assuming we have a new endpoint that fetches `EndUser`s, how will you make the query using the read replica?